src/rules/selector-no-empty/index.js

```diff
@@ -0,0 +1,37 @@
+import {
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-no-empty"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: "Unexpected empty selector",
+})
+
+export default function (actual) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual })
+    if (!validOptions) { return }
+
+    root.walkRules(rule => {
+      let index = 0
+
+      rule.selector.split(",").forEach(item => {
+        index += item.length + 1
+
+        if (item.trim()) { return }
```

Comment:
Let's be explicit that `item.trim() !== ''`. That will make the intention more clear.